### PR TITLE
[django] Add support for django 2.2 #231

### LINKED ISSUE #231: [django] Add support for django 2.2
Add support for django 2.2.

### PR DESCRIPTION
**Pull request by Vivekrajput20**
closes #231

### CODE REVIEW

File: django_freeradius/tests/base/test_admin.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def test_radiusgroupreply_change(self):
                        op=':=', value='PPP')
         obj = self._create_radius_groupreply(**options)
         response = self.client.get(reverse('admin:{0}_radiusgroupreply_change'.format(self.app_name),
-                                   args=[obj.pk]))
+                                           args=[obj.pk]))
```
Comment (nemesisdesign) [inline django_freeradius/tests/base/test_admin.py]: can you revert this change please?

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-django>=2.0,<2.2
+django>=2.2,<2.2.1
```
Comment (nemesisdesign) [inline requirements.txt]: why are you making django 2.2 the minimum version? This shouldn't be done unless we decide so, but we didn't discuss this. Keep in mind for the future.

Comment (Vivekrajput20) [reply requirements.txt]: I'll keep that in mind, also do I need to change it or is it okay for now?

Comment (nemesisdesign) [reply requirements.txt]: please revert this change
